src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -230,5 +237,18 @@ bool MulticopterLandDetector::_get_landed_state()
 	return true;
 }
 
+float MulticopterLandDetector::get_takeoff_throttle()
+{
+	if (_control_mode.flag_control_manual_enabled && _control_mode.flag_control_position_enabled) {
+		return 0.9f;
+	}
+
+	if (_control_mode.flag_control_manual_enabled && _control_mode.flag_control_attitude_enabled) {
+		return 0.15f;
+	}
+
+	return 0.0f;
+}
```

Comment:
can you specify what 0.9 and 0.15 is and where the number come from. I realize that 0.15 was already used before, but since this value somewhat represent min and max, we could also label them instead of magic numbers